activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableCreatedEventHandler.java

```diff
@@ -72,8 +72,8 @@ private ProcessVariableEntity createProcessVariableEntity(CloudVariableCreatedEv
                                                                          new Date(variableCreatedEvent.getTimestamp()),
                                                                          new Date(variableCreatedEvent.getTimestamp()),
                                                                          null);
-        variableEntity.setValue(variableCreatedEvent.getEntity()
-                                                    .getValue());
+        variableEntity.setValue(variableCreatedEvent.getEntity().getValue());
+        variableEntity.setDefinitionId(variableCreatedEvent.getVariableDefinitionId());
```

Comment:
as for the field, can we call it VariableDefinitionId

Reply:
Done